CHANGELOG.md

```diff
@@ -2,6 +2,59 @@
 
 Notable changes to QMCPACK will be documented in this file.
 
+## [3.5.0] - 2018-08-02
+
+### Notes
+
+This release includes support for the latest Quantum Espresso version 6.3,
+an initial implementation of periodic Gaussian support via PySCF, and a new
+version of the hybrid or "APW" representation of orbitals. Many minor
+bugs have been fixed, configuration and documentation improved. 
+
+Note that the PDF manuals are no longer included with the source. Versions
+are available online via https://qmcpack.org . The PDFs can be built
+using manual/build_manual.sh and
+nexus/documentation/user_guide_source/build_nexus_user_guide.sh
+
+Attention developers: This version contains substantially fewer source
+lines than previous versions due to clean out of old code and unused
+execution paths. Refactoring to improve the internal structure of
+QMCPACK is ongoing. Track the develop branch and follow discussion on
+GitHub closely to avoid difficult merges.
+
+* Support for Quantum Espresso 6.3 and 6.2.1. Check documentation to
+  ensure compiled with required HDF5 support.
+* Support for periodic gaussians and PySCF generated
+  wavefunctions. Initial version is limited to Gamma-point.
+* Improved hybrid representation of single particle orbitals
+  (APW-like) for significantly reduced memory usage and possible
+  accuracy increase compared to conventional spline representation.
```

Comment:
Could you add an arxiv pointer here? arXiv:1805.07406

Reply:
Done